assignments/arrays.js

```diff
@@ -63,32 +74,54 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
```

Comment:
I like seeing you use for loops. Remember that Arrays begin with 0 instead of 1, car 33 would actually be in the 32nd position in an array so you would want to add a -1. Let me know if you want me to go over this. Also since we already know the specific car we want, we could just console.log like this console.log(`Car 33 is a ${inventory[32].car_year} ${inventory[32].car_make} ${inventory[32].car_model}` ); When I did this assignment I also used a for loop even though it is not needed.